### ISSUE TITLE
Cartliverly/botania minecart incompatibility 

### ISSUE THREAD

**Issue by dahjin** [kuenzign/CartLivery#2]
i'm unable to Color the Botania Mana pool minecart's with Cartlivery's mod i am unsure who to ask for a fix so im sending this to both githubs.

**Comment by kuenzign** [comment]
Are you using Railcraft?

**Comment by dahjin** [comment]
Yes, it works on regular carts and railcraft carts but botania adds its own cart i believe. it applies to the cart but it only shows up in waila not in world.

**Comment by liach** [comment]
So @kuenzign needs to add an override method of botania cart rendering.

**Comment by kuenzign** [comment]
I don't want to raise expectations as I still so much left to do and I don't have much free time, but I wanted to show off this:

![2020-04-13_23 18 03](https://user-images.githubusercontent.com/5274184/79182652-4fe6ed80-7ddd-11ea-91d4-273fc26a0868.png)
